source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -2518,38 +2501,64 @@ export class ImageSet extends CardElementContainer {
         let element: HTMLElement | undefined = undefined;
 
         if (this._images.length > 0) {
+            const imageSetIsGrid = (this.presentationStyle === Enums.ImageSetPresentationStyle.Grid);
```

Comment:
do we need version check here since renderer can work with different version of Enums?

Reply:
I believe that should be enforced here:

https://github.com/microsoft/AdaptiveCards/blob/67f7109761bd6b4fdf565b121fc7c64472f3d0bc/source/nodejs/adaptivecards/src/card-elements.ts#L2472-L2477